MessageViewController/MessageTextView.swift

```diff
@@ -15,30 +15,48 @@ public protocol MessageTextViewListener: class {
 
 open class MessageTextView: UITextView, UITextViewDelegate {
 
-    private let placeholderLabel = UILabel()
-
-    private var listeners: NSHashTable<AnyObject> = NSHashTable.weakObjects()
+    internal let placeholderLabel = UILabel()
+    internal var listeners: NSHashTable<AnyObject> = NSHashTable.weakObjects()
 
     open override var delegate: UITextViewDelegate? {
         get { return self }
         set {}
     }
 
-    open var defaultFont: UIFont? = UIFont.preferredFont(forTextStyle: .body)
+    open var defaultFont = UIFont.preferredFont(forTextStyle: .body) {
+        didSet {
+            defaultTextAttributes[NSAttributedStringKey.font.rawValue] = defaultFont
+        }
+    }
 
-    open var defaultTextColor: UIColor? = .black
+    open var defaultTextColor = UIColor.black {
+        didSet {
+            defaultTextAttributes[NSAttributedStringKey.foregroundColor.rawValue] = defaultTextColor
+        }
+    }
+
+    internal var defaultTextAttributes: [String: Any] = {
```

Comment:
Why is this a `[String: Any]` rather than an `[NSAttributedStringKey: Any]` now?

Reply:
Matching `typingAttributes: [String: Any]`. There's something wrong w/ `typingAttributes`'s type: [see docs](https://developer.apple.com/documentation/uikit/uitextview/1618629-typingattributes)